manual/source/nxdl-types.rst

```diff
@@ -16,7 +16,8 @@ describe the expected type of data for a NeXus field. These terms are very
 broad. More specific terms are used in actual NeXus data files that describe
 size and array dimensions. In addition to the types in the following table, the
 ``NAPI`` type is defined when one wishes to permit a field
-with any of these data types.
+with any of these data types. The default type ``NX_CHAR`` is applied in cases 
+where a field is defined in an NXDL specification without explicit assignment of a type.
```

Comment:
where the `type` of a field or attribute

Reply:
done in new commit